### DISCUSSION TITLE
User Login and authentication management

### DISCUSSION THREAD

**Discussion by rahulvemula** [rahulvemula/SE-leasingPortal#124] [Q&A] [answered]
How are we handling user login?
- Can we have the documentation for the new login api? The request and response formats.
- How do we pass the token in the further api calls? In header or cookie? And in which key value format?

@vamsibvv @mitulmandaliya

**Comment by mitulmandaliya** [comment, marked as answer]
1) Login API is a POST request with 2 parameters in body. username and password
2) token should be passed in header with "token" as the key.
